docs/manual-curation/README.md

```diff
@@ -0,0 +1,40 @@
+# Manual trait name curation protocol
+_Issue template: https://www.ebi.ac.uk/panda/jira/browse/EVA-1911_
+
+## Introduction
+Data which we submit to Open Targets represents associations between traits and variants. In the case of ClinVar, traits are almost always inherited diseases. In the Open Targets evidence strings, they must be represented using terms from the [Experimental Factor Ontology](https://www.ebi.ac.uk/efo).
+
+Mapping free-text trait names to ontology terms is the only part of the Open Targets submission process which cannot be automated, and this is the reason it is contained as a separate protocol.
+
+The idea is to run this protocol periodically and independently of the main submission protocol. While at submission time Open Targets data is always synchronised to an older, fixed ClinVar release, all new ClinVar data will be *eventually* incorporated into future Open Targets releases. Hence, it makes sense to do the curation work ahead of time and to decouple this process from the main evidence string generation.
+
+Detailed description of the current approach can be found [in the addendum.](detailed-description.md) This protocol, which is currently performed with TSV files and spreadsheets, will be replaced by a web interface for manual curation before the end of 2020.
+
```

Comment:
When is this protocol initiated. If it is to be desynchronised from the OpenTarget submission then it needs to have its own schedule.

Reply:
Yes, that's true. The schedule itself can be for example every two months, offset by one month from the Open Targets core schedule. E. g. in May we are working on Open Targets, then in June on curation, in July on OT again, to split the load evenly.

I'm not sure what would be the best way to periodically remind us to do the curation. Do we have any other periodic activities like that? Could we just add a point to the sprint meeting template to not forget about this?